simdev/main.py

```diff
@@ -0,0 +1,121 @@
+import logging
```

Comment:
why don't you keep `__main__.py`? It allows to skip `main` during the call of the module `python3 -m simdev.main`? If you keep rename this file to `__main__.py` - you can call it `python3 -m simdev`

Reply:
Renamed!